.gitignore

```diff
@@ -154,6 +155,7 @@ integration_tests/scripts/main.tf
 integration_tests/scripts/.terraform
 .DS_Store
 snmp-ssh-key.pem
+new_integration_testing
```

Comment:
is it created while running tests?

Reply:
Oh sorry, `new_integration_testing` was a temporary directory, I'll delete this entry from .gitignore